pywbemtools/_click_extensions.py

```diff
@@ -1,47 +1,50 @@
 """
-This file contains extensions to Click specifically for pywbemcli
+This file contains extensions to Click specifically for pywbemtools.
```

Comment:
Lets get rid of the word specifically.  It is redundant. My original error

Reply:
DONE